client/flutter/lib/components/home_page_sections/home_page_protect_yourself.dart

```diff
@@ -79,23 +78,73 @@ class _HomePageProtectYourself extends State<HomePageProtectYourself> {
 
   List<Widget> _buildCards() {
     final screenWidth = MediaQuery.of(context).size.width;
-    final TextStyle normalText = TextStyle(
-      color: Constants.neutralTextDarkColor,
-      fontSize: 12 * MediaQuery.textScaleFactorOf(context),
-      height: 1.33,
-    );
     return (_factContent?.items ?? [])
         .map((fact) => SizedBox(
               width: screenWidth * 0.75,
               child: Padding(
-                padding: EdgeInsets.symmetric(horizontal: 6.0),
-                child: ProtectYourselfCard.fromFact(fact,
-                    defaultTextStyle: normalText,
-                    shouldAnimate: false,
-                    borderRadius: BorderRadius.circular(8.0),
-                    childBackgroundColor: Constants.illustrationBlue1Color),
+                padding: EdgeInsets.symmetric(horizontal: 10.0),
+                child: _HomeProtectYourselfCard(fact: fact),
               ),
             ))
         .toList();
   }
 }
+
+class _HomeProtectYourselfCard extends StatelessWidget {
+  final FactItem fact;
+
+  const _HomeProtectYourselfCard({
+    @required this.fact,
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(16.0),
+      child: Container(
+        color: Constants.primaryDarkColor,
+        child: Column(
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.fromLTRB(28.0, 36.0, 28.0, 20.0),
+              child: _buildBody(context, this.fact.body),
+            ),
+            Expanded(
+              child: Container(
+                alignment: Alignment.bottomCenter,
+                child:
+                    SvgPicture.asset('assets/svg/${this.fact.imageName}.svg'),
+              ),
+            )
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildBody(BuildContext context, String body) {
+    final TextStyle defaultTextStyle = TextStyle(
```

Comment:
Think we should have these HTML text styles generated by your theme/variant class (with the text scale factory)